VAMobile/src/screens/HealthScreen/Pharmacy/RefillScreens/RefillRequestSummary.tsx

```diff
@@ -80,28 +80,28 @@ function RefillRequestSummary({ navigation, route }: RefillRequestSummaryProps)
   })
 
   const renderAlert = (): ReactElement => {
-    let alertBoxProps: AlertBoxProps
+    let alertBoxProps: AlertWithHapticsProps
     switch (status) {
       case REQUEST_STATUS.SUCCESS:
         alertBoxProps = {
-          border: 'success',
-          title: t('prescriptions.refillRequestSummary.success'),
+          variant: 'success',
+          header: t('prescriptions.refillRequestSummary.success'),
         }
         break
       case REQUEST_STATUS.MIX:
       case REQUEST_STATUS.FAILED:
       default:
         alertBoxProps = {
-          border: 'error',
-          title: t('prescriptions.refillRequestSummary.mix', { count: requestFailed.length }),
-          text: t('prescriptions.refillRequestSummary.tryAgain'),
-          textA11yLabel: a11yLabelVA(t('prescriptions.refillRequestSummary.tryAgain')),
+          variant: 'error',
+          header: t('prescriptions.refillRequestSummary.mix', { count: requestFailed.length }),
+          description: t('prescriptions.refillRequestSummary.tryAgain'),
+          descriptionA11yLabel: a11yLabelVA(t('prescriptions.refillRequestSummary.tryAgain')),
         }
         break
     }
     return (
       <Box mb={theme.dimensions.standardMarginBetween}>
-        <AlertBox {...alertBoxProps}>
+        <AlertWithHaptics {...alertBoxProps}>
           {status !== REQUEST_STATUS.SUCCESS && (
             <Box mt={theme.dimensions.standardMarginBetween}>
               <Button
```

Comment:
Could potentially be simplified moving this to the `primaryButton` or `secondaryButton` props to AlertWithHaptics.